### ISSUE TITLE
IFF column renaming

### ISSUE THREAD

**Issue by jmcfarland-swri** [jmcfarland-swri/NASA_ULI_InfoFusion#3]
@edecarlo-swri Please review the column renaming below.  I attempted to match the behavior of the original readIFF code.  The one that looks suspicious is `'groundSpeed':'tas'`.

https://github.com/jmcfarland-swri/NASA_ULI_InfoFusion/blob/534f48814236716dca436368b5237c0194f10a60/src/PARA_ATM/io/iff.py#L102-L110

**Comment by edecarlo-swri** [comment]
Currently waiting on info from ATAC specifically on `groundSpeed:'tas`. It is unclear what groundSpeed refers to in the IFF dataset for aircraft enroute and whether or not this is actually equivalent to total air speed (TAS). Note that NATS uses TAS interchangeably with speed for aircraft on the ground.

**Comment by edecarlo-swri** [comment]
Also, in the IFF dataset the units of `alt` are in 100s of feet which would actually be considered a flight level (FL). NATS altitude results, however, are reported in feet. I think we should either multiply the IFF reported altitudes by 100 to be consistent or rename the `alt` to `flightLevel`.

**Comment by jmcfarland-swri** [comment]
OK, I went ahead and converted `altitude` (the renamed column) to feet (daacf38).

**Comment by jmcfarland-swri** [comment]
Per discussion with John Schade, `groundSpeed` really is ground speed.  Also, `course` is the actual direction the aircraft is taking, whereas `heading` (which would typically not be available in the IFF data) is the direction it is pointed.

Probably what needs to be done is that we should not rename these two columns like we are currently doing, since the names would then be incorrect.  However, plotting code will need to be updated to handle the potentially different available data (e.g. `groundSpeed` or `tas` may be available, and similarly for `heading` and `course`).

**Comment by edecarlo-swri** [comment]
I agree with your recommendation on not renaming the columns and also on how to handle it in the plotting code.
